ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tunnel-interfaces.j2

```diff
@@ -0,0 +1,79 @@
+{% if tunnel_interfaces is arista.avd.defined %}
+
+## Tunnel Interfaces
+
+### Tunnel Interfaces Summary
+
+| Interface | Description | VRF | MTU | Shutdown |
+| --------- | ----------- | --- | --- | -------- |
+{%     for tunnel_interface in tunnel_interfaces | arista.avd.natural_sort('name') %}
+{%         set description = tunnel_interface.description | arista.avd.default('-') %}
+{%         set vrf = tunnel_interface.vrf | arista.avd.default('default') %}
+{%         set mtu = tunnel_interface.mtu | arista.avd.default('-') %}
+{%         set shutdown = tunnel_interface.shutdown | arista.avd.default('-') %}
+| {{ tunnel_interface.name }} | {{ description }} | {{ vrf }} | {{ mtu }} | {{ shutdown }} |
+{%     endfor %}
+{# IPv4 #}
+{%     set tunnel_interfaces_ipv4 = [] %}
+{%     for tunnel_interface in tunnel_interfaces | arista.avd.default([]) %}
+{%         if tunnel_interface.ip_address is arista.avd.defined %}
+{%             do tunnel_interfaces_ipv4.append(tunnel_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if tunnel_interfaces_ipv4 | length > 0 %}
+
+#### IPv4
+
+| Interface | VRF | IP Address | TCP MSS | TCP MSS Direction | Source Interface | Destination | PMTU-Discovery | ACL In | ACL Out |
```

Comment:
move source, destination and pmtu to the general table.

Reply:
Addressed in 0b8ddf0